orcestra_book/_ext/reports.py

```diff
@@ -81,32 +81,73 @@ def run(self):
         return nodes.raw(text=fm[self.text]), []
 
 
-def collect_frontmatter(src):
-    flights = (src / "flight_reports").glob("*[0-9]*[a-z].md")
-    func = partial(load_frontmatter, derive_flight=True)
+class LogoRole(SphinxRole):
+    """Add a small campaign logo to the upper-right corner of a page."""
+    def run(self):
+        src = pathlib.Path(self.env.srcdir)
+
+        logo_path = list((src / "logos").glob(f"*_{self.text}.svg"))
```

Comment:
This may result in `logo_path` being an absolute path on the building machines local filesystem. Before entering this path in some HTML context, the path needs to be converted, ideally to a path relative to the location where the current Role is being used.